src/main/java/core/basesyntax/figure/Circle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.interfaces.AreaCalculator;
+
+public class Circle extends Figure implements AreaCalculator {
```

Comment:
```suggestion
public class Circle extends Figure {
```
the same for other figures